src/main/scala/org/zalando/spark/jsonschema/SchemaConverter.scala

```diff
@@ -33,6 +33,7 @@ object SchemaConverter {
   val typeMap = Map(
     "string" -> StringType,
     "number" -> DoubleType,
+    "float" -> FloatType,
```

Comment:
@AndreiDev thanks for the pull request, it is nice to know the library seems useful. One thing: could you add this to the test in SchemaConverterTest. Thanks